packages/table/preview/index.tsx

```diff
@@ -246,6 +252,12 @@ class MutableTable extends React.Component<{}, IMutableTableState> {
     }
 
     public componentWillUpdate(_nextProps: {}, nextState: IMutableTableState) {
+        // save off any changes to the state to local storage
+        Object.keys(nextState).forEach((stateKey: keyof IMutableTableState) => {
+            if (this.state[stateKey] !== nextState[stateKey]) {
```

Comment:
nit: With `if` statements, compare the new thing to the old/fixed-value thing in left-to-right order:

`if (newThing !== oldThing)`
`if (newThing !== null)`
`if (newThing !== 100)`

So in this case, I prefer `if (nextState[stateKey] !== this.state[stateKey])`.

Reply:
I do totally agree on the second two, but the first one I'm amvivalent on. No strong feelings either way, and I assume since you've been writing all the code so it's standard, so I'll follow suit. Curious _why_ the ordering preference, though.